packages/react-button/src/components/Button/Button.tsx

```diff
@@ -5,8 +5,35 @@ import { renderButton } from './renderButton';
 import { useButtonStyles } from './useButtonStyles';
 
 /**
- * Define a styled Button, using the `useButton` hook.
- * {@docCategory Button}
+ * Buttons give people a way to trigger an action.
```

Comment:
I don't see this content anywhere on the docs page for `Button`, is it supposed to appear there ?

Reply:
it only works when running Storybook for individual package - I’ve added warning to the PR description